docs/production/scim.md

```diff
@@ -1,34 +1,65 @@
+Zulip has beta support for user provisioning and deprovisioning via
+the SCIM protocol. In SCIM, a third-party SCIM Identity Provider (IdP)
+acts the SCIM client, connecting to the service provider (your Zulip
+server).
 
-Groups are currently not supported.
+See the [SCIM help center page](https://zulip.com/scim) for
+documentation on SCIM in [Zulip Cloud](https://zulip.com) as well as
+detailed documentation for how to configure some SCIM IdP providers.
+
+Synchronizing groups via SCIM is currently not supported.
 
 ## Server configuration
 
-The server-side configuration is straightforward:
+The Zulip server-side configuration is straightforward:
+
+1. Pick a client name for your SCIM client. This name is internal to
+   your Zulip configuration, so the name of your IdP provider is a
+   good choice. We'll use `okta` in the examples below.
+1. First a SCIM client entry needs to be added to the database. Run
+   `manage.py add_scim_client <client name> -r <subdomain>`. For
+   example, if your organization is hosted at on a subdomain
+   (`subdomain.zulip.example.com`):
+
+   ```
+   /home/zulip/deployments/current/manage.py add_scim_client okta -r 'subdomain'
+   ```
+
+   Or your organization is hosted on the root domain (`zulip.example.com`):
+
+   ```
+   /home/zulip/deployments/current/manage.py add_scim_client okta -r ""
+   ```
+
+   See the [management command documentation](./management-commands.md)
+   for details on how to run management commands.
+
+1. Configure the Zulip server by adding an `SCIM_CONFIG` block to your
+   `/etc/zulip/settings.py`:
 
-1. First a SCIM client entry needs to be added to the database.
-   Run `manage.py add_scim_client <your client name> -r <your organization's subdomain>`.
-   For example, `manage.py add_scim_client my-okta-client -r ''` if your organization
-   is hosted on the root subdomain.
-1. Add `SCIM_CONFIG` to your `/etc/zulip/settings.py`:
    ```
    SCIM_CONFIG = {
-        "your organization's subdomain": {
+        "subdomain": {
             "bearer_token": "<secret token>",
-            "scim_client_name": "your-client-name",
+            "scim_client_name": "okta",
             "name_formatted_included": False,
         }
     }
    ```
-   The `bearer_token` should contain a secure, secret token that you generate. The SCIM
-   client will authenticate its requests to the server by putting it in the `WWW-Authenticate`
-   header like this: `WWW-Authenticate: Bearer <secret token>`.
-   `name_formatted_included` needs to be set to `False` for Okta. It tells Zulip
-   whether the client includes `name.formatted` in its `User` representation.
-1. Now you can proceed to [configuring the client](http://zulip.com/help/scim).
-   Use the value `Bearer <secret token>` using the `bearer_token` you've generated earlier
-   as the `API token` Okta will ask for when configuring authentication details.
+
+   The `bearer_token` should contain a secure, secret token that you
+   generate.
```

Comment:
Do we have a recommended command to generate these? I feel like a lot of folks may get stuck at this step.

Reply:
This is essentially just password generation. I expanded the paragraph:
```
   The `bearer_token` should contain a secure, secret token that you
   generate. You can use any secure password generation tools for this,
   such as the `apg` command included by default in some Linux distributions.
   For example, `apg -m20` will generate some passwords of minimum length 20
   for you.
```

Reply:
Do we have rate-limiting on this endpoint? If not, we might want to suggest more entropy.

Reply:
Please reply to this in #backend, since I'm about to merge this PR, and I think we can fix this forward if we determine we need something stricter. (Seems easy to adjust in a follow-up).